integrations/langflow/examples/basic_conversion.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+"""Basic conversion example for Langflow to Stepflow integration."""
+
+import json
+from pathlib import Path
+
+from stepflow_langflow_integration import LangflowConverter
+
+
+def main():
```

Comment:
Does this example add value? Or should we just update the redame to talk about how to use the CLI?